plugins/module_utils/config_utils.py

```diff
@@ -435,8 +435,14 @@ def set(self, value: str, setting: str) -> None:
             _setting: Element = self._config_xml_tree.find(xpath)
 
         # If the element is present we will verify it's .text value
-        elif _setting.text in [None, "", " "]:
-            raise NotImplementedError("Currently only text settings supported")
+        elif _setting.text is None or _setting.text.strip() == "":
+            # check if setting has children
+            if len(_setting.items()) > 0:
```

Comment:
Sorry for the follow up, I've missed that in the first review. I think here we can just simply cast the Element to a list since  lists are not considered truthy if they are empty
```suggestion
            if list(_setting):
```